jobs/SprintRelease/Jenkinsfile

```diff
@@ -42,6 +43,8 @@ node{
                 if(JUMP_VERSION == "true"){
                     load("jobs/SprintRelease/bump_version.groovy")
                 }
+                
+                  submodule.submodule("update");
```

Comment:
If the submodule is updated, the RackHD will have a new commit, right?
If so, the commit of RackHD in manifest is not the latest commit.
The tested RackHD is still the version without updating submodules.
Why not update submodules after unit test and function test?